recipes/openmesh/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_APPS=OFF \
+  -DOPENMESH_PYTHON_VERSION=${PY_VER} \
+  -DOPENMESH_BUILD_PYTHON_UNIT_TESTS=ON \
+  ..
```

Comment:
So normally one would do...

```
cmake ...
make
make check <or> ctest
make install
```

Though I'm not seeing `make`. Is it lumped in with `make install`?